build/release.sh

```diff
@@ -38,4 +38,10 @@ fi
 
 kube::build::copy_output
 kube::release::package_tarballs
+# If we have these variables set then we want to build all docker images.
+# Since regular build functions above don't build hyperkube we need to do
+# it manually.
+if [[ -n "${KUBE_DOCKER_IMAGE_TAG-}" && -n "${KUBE_DOCKER_REGISTRY-}" ]]; then
```

Comment:
Apologies in advance for the naive questions.
Is there a better place for this?  Could this go into kube::build::docker_build or another function that builds a list of passed in cluster/images directories?  I don't know know all of the relationships yet, so I don't know what is appropriate.